joint_limits/src/joint_limiter_interface.cpp

```diff
@@ -1,72 +0,0 @@
-// Copyright (c) 2021, Stogl Robotics Consulting UG (haftungsbeschränkt)
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-/// \author Denis Stogl
-
-#include "joint_limits/joint_limiter_interface.hpp"
-
-#include <string>
-#include <vector>
-
-#include "joint_limits/joint_limits_rosparam.hpp"
-
-// TODO(anyone): Add handing of SoftLimits
-namespace joint_limits
-{
-template <>
-bool JointLimiterInterface<JointLimits>::init(
```

Comment:
I am not sure why did you remove this file? It's very useful to have default implementations.

Reply:
The file is not necessary and I thought `simple_joint_limiter` would be a good example for others to follow.